db/blip_sync_context.go

```diff
@@ -225,7 +225,7 @@ func (bsc *BlipSyncContext) copyContextDatabase() *Database {
 }
 
 func (bsc *BlipSyncContext) _copyContextDatabase() *Database {
-	databaseCopy, _ := GetDatabase(bsc.blipContextDb.DatabaseContext, bsc.blipContextDb.User())
+	databaseCopy, _ := GetDatabase(context.TODO(), bsc.blipContextDb.DatabaseContext, bsc.blipContextDb.User())
```

Comment:
_copyContextDatabase gets called a lot (once per user-scoped blip handler, I think?).  I don't think we'd want to create a new context and add db log context to it (in GetDatabase) every time.  This might go back to the question of whether Database should still store a ctx in its struct, but I thought we'd want to pass bsc.loggingCtx here.

Reply:
well, I just flagged it for now, as we talked for changing APIs, and eval what we can pass here later, when we get to /db package (and eval how bsc.loggingCtx is being set)... but I can change it now if you're sure of it, let me know.